src/include/CmdArgsHelper.h

```diff
@@ -20,6 +20,7 @@ const char *cmdArgsStrAquarium = R"(Options and arguments:
 --enable-full-screen-mode       : Render aquarium in full screen mode instead of window mode.
 --integrated-gpu        : Choose integrated gpu to render the application. This is only supported on Dawn and D3D12 backend.
 --num-fish [count]      : specifies how many fishes will be rendered.
+--print-average-fps     : print avarage fps when exit the application. 

```

Comment:
Is there any possibility to add more performance data besides average fps in the future? If so, something like `--print-benchmark` could be more flexible.

Reply:
Yes, I think so. I prefer to name it '--print-log'.